src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -7,11 +7,14 @@
 
 // Types moved down into System.Runtime.Handles
```

Comment:
Nit: This comment is not correct. All these types are in System.Runtime

Reply:
I see. Double hop to System.Runtime it looks like: https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Handles/ref/System.Runtime.Handles.Forwards.cs#L9